tests/Resources/test-domains.py

```diff
@@ -0,0 +1,32 @@
+"""
+See if our domains work
+"""
+import json
+from checkStatusCode import check_status_code
+
+def test_domains(file_path="../../domains.json"):
```

Comment:
Är det här ett bra sätt att hämta domänerna? Man skulle kanske ha alternativet att skicka in en array eller nåt med domäner också?